plugin.json

```
@@ -1,9 +1,9 @@
 {
-    "id": "com.mattermost.plugin-starter-template",
-    "name": "Plugin Starter Template",
-    "description": "This plugin serves as a starting point for writing a Mattermost plugin.",
+    "id": "com.mattermost.agenda-plugin",
```

I don't think anyone has agreed on names, but most of the MM plugins are not namespaced like this.

---

What is the convention? I followed the example that starter template originally has as well as Demo Plugin and Custom Attributes. 

I personally don't like it ending in `-plugin`. How about 
```suggestion
    "id": "com.mattermost.agenda",
```

---

I like @marianunez's proposal.  I'll use the consensus choice moving forward, while doing the other plugin reviews.  

`repo-name:` mattermost-plugin-<name-with-dashes>
`id:` com.mattermost.<name-with-dashes>

---

@crspeller our current plugin `id` [documentation](https://developers.mattermost.com/extend/plugins/manifest-reference/#id) does reference a reverse DNS notation.

Just bringing this up so it can be updated to what the consensus ends up moving forward.

---

As I understand @crspeller comment, `com.mattermost.agenda` is good.

---

Yup. That's fine.